### ISSUE TITLE
[Issue]: Error after enabling "VAE rollback feature in case of NaNs"

### ISSUE THREAD

**Issue by YKefasu** [vladmandic/automatic#455]
### Issue Description

Error after enabling "VAE rollback feature in case of NaNs"
how can i disable it back?

```
╭───────────────────────────────────────── Traceback (most recent call last) ──────────────────────────────────────────╮
│ C:\New folder\automatic\launch.py:101 in <module>                                                                    │
│                                                                                                                      │
│   100     import webui                                                                                               │
│ ❱ 101     webui.webui()                                                                                              │
│   102                                                                                                                │
│                                                                                                                      │
│ C:\New folder\automatic\webui.py:216 in webui                                                                        │
│                                                                                                                      │
│   215 def webui():                                                                                                   │
│ ❱ 216     start_ui()                                                                                                 │
│   217     load_model()                                                                                               │
│                                                                                                                      │
│ C:\New folder\automatic\webui.py:169 in start_ui                                                                     │
│                                                                                                                      │
│   168     create_paths(opts)                                                                                         │
│ ❱ 169     initialize()                                                                                               │
│   170     if shared.opts.clean_temp_dir_at_start:                                                                    │
│                                                                                                                      │
│ C:\New folder\automatic\webui.py:78 in initialize                                                                    │
│                                                                                                                      │
│    77 def initialize():                                                                                              │
│ ❱  78     check_rollback_vae()                                                                                       │
│    79                                                                                                                │
│                                                                                                                      │
│ C:\New folder\automatic\webui.py:69 in check_rollback_vae                                                            │
│                                                                                                                      │
│    68     if shared.cmd_opts.rollback_vae:                                                                           │
│ ❱  69         if version.parse(torch.__version__) < version.parse('2.1'):                                            │
│    70             print("If your PyTorch version is lower than PyTorch 2.1, Rollback VAE will                        │
╰──────────────────────────────────────────────────────────────────────────────────────────────────────────────────────╯
NameError: name 'version' is not defined
Press any key to continue . . .
```

**Comment by Gazzoo-byte** [comment]
Original issue is resolved as of 20b64aad however I have got the following upon trying to load a VAE

```
changing setting sd_vae to blessed2.vae.pt: NameError
 Traceback (most recent call last) 
 E:\AIStuff\automatic\modules\shared.py:526 in set                                                                                                                                                    

   525          try:                                                                                                                                                                               
 ❱ 526             self.data_labels[key].onchange()                                                                                                                                               
   527          except Exception as e:                                                                                                                                                             

 E:\AIStuff\automatic\modules\call_queue.py:16 in f                                                                                                                                                   

    15       with queue_lock:                                                                                                                                                                       
 ❱  16          res = func(*args, **kwargs)                                                                                                                                                        
    17                                                                                                                                                                                                

 E:\AIStuff\automatic\webui.py:105 in <lambda>                                                                                                                                                        

   104                                                                                                                                                                                               
 ❱ 105    shared.opts.onchange("sd_vae", wrap_queued_call(lambda: modules.sd_vae.reload_vae_we                                                                                                       
   106    shared.opts.onchange("sd_vae_as_default", wrap_queued_call(lambda: modules.sd_vae.re                                                                                                       

 E:\AIStuff\automatic\modules\sd_vae.py:200 in reload_vae_weights                                                                                                                                     

   199    sd_hijack.model_hijack.undo_hijack(sd_model)                                                                                                                                               
 ❱ 200    if shared.cmd_opts.rollback_vae and devices.dtype_vae == torch.bfloat16:                                                                                                                   
   201       devices.dtype_vae = torch.float16                                                                                                                                                      

NameError: name 'torch' is not defined
```

**Comment by vladmandic** [comment]
i don't know how i missed that. fixed.
